src/components/FilterTop.js

```diff
@@ -375,18 +385,20 @@ function Filter(props) {
                 onChange={(_oldVal, newVal) => {
                   //triggered by onInputChange
                   console.log('newVal -- ', newVal);
-                  if (newVal && newVal.tagName === 'Not set') {
-                    setTag('Not set');
-                  } else {
-                    setTag(newVal);
-                  }
+                  setTag(newVal);
                 }}
                 onInputChange={(_oldVal, newVal) => {
                   setTagSearchString(newVal);
                 }}
                 renderInput={(params) => {
                   return <TextField {...params} label="Tag" />;
                 }}
+                // Option B for divider
```

Comment:
If Material UI doesn't provide a good way of adding dividers, let's go without for now.